### PR TITLE
Weekly Report: add PayPal donations

### PR DESCRIPTION
**Pull request by HipsterBrown**
Example output:

```
hipsterbrown:api (master) $ MANUAL=true START_DATE=2018-07-14 ./node_modules/.bin/babel-node cron/weekly/slack-report.js
Connecting to postgres://127.0.0.1/opencollective_prod_snapshot
Weekly activity summary (excluding OC team):

* Donations:
  - 212 (decrease of -2664) donations received totaling:
    * 196 EUR (decrease of -1814)
    * 405 GBP (decrease of -125)
    * 17806 USD (decrease of -48088)
  - 12 (decrease of -7) paypal donations received totaling:
    * 152 USD (decrease of -890)
* Expenses:
  - 9 pending expenses totaling:
    * -1694 EUR
    * -112 GBP
    * -1347 USD
  - 22 approved expenses totaling:
    * -592.68 EUR
    * -4.74 GBP
    * -1859 MXN
    * -676.08 USD
  - 9 rejected expenses totaling:
    * -1601.27 EUR
    * -4805.14 USD
  - 58 (decrease of -4) paid expenses totaling:
    * -172.05 EUR (decrease of -2122)
    * -245.25 GBP (increase of 245)
    * -1120 MXN (increase of 1120)
    * -37209.1 USD (decrease of -11926)
* Collectives:
  - 138 (decrease of -218) active collectives
  - 15 (increase of 0) new collectives:
    * ahcodes (C-corp,association)
    * humanitarian-ai-meetup (open source)
    * social-platform-donut (open source)
    * cyclonic-games (C-corp,other)
    * reakit (open source)
    * mint (open source)
    * noplasticapp (worldpeacenow)
    * sit (open source)
    * curso-lenguaje-de-signos (C-corp,coop)
    * 400-toits-daken (#brusselstogether)
    * react-svg-loader (open source)
    * torontojs (javascript,javascript-applications,javascript-frameworks,javascript-libraries,web,web-development,softwaredev,functional-programming,opensource,programming-languages,computer-programming,nodejs,front-end-development,html5,meetup,Tech meetups)
    * tech4good-pleasanton (information-technology,technology,nonprofit,netsquared,nonprofit-networking,socialnetwork,social-good,computer-programming,web,professional-networking,personal-development,community-building,entrepreneurship,projectmgmt,education,meetup,Tech meetups, techsoup)
    * cryptpad (open source)
    * publii (open source)

Weekly reporting done!
```

### CODE REVIEW

File: cron/weekly/slack-report.js [hunk 1]
```diff
@@ -230,6 +255,8 @@ function reportString({
 * Donations:
   - ${donationCount} (${compareNumbers(donationCount, priorDonationCount)}) donations received totaling:
     ${donationAmount.map(({ SUM, currency }) => `* ${SUM/100} ${currency} (${compareNumbers(SUM/100, getSum(priorDonationAmount, currency)/100)})`).join('\n    ')}
+  - ${paypalDonationCount} (${compareNumbers(paypalDonationCount, priorPaypalDonationCount)}) paypal donations received totaling:
+    * ${paypalDonationAmount/100} USD (${compareNumbers(paypalDonationAmount/100, priorPaypalDonationAmount/100)})
```
Comment (xdamman) [inline cron/weekly/slack-report.js]: can you use `utils.formatCurrency(amount, currency)` to properly format this?

**Comment by xdamman** [comment]
Also, can you remove `decrease of`? It's redundant with the negative number that comes after. Just having `-100` or `+100` is enough (and concise).

**Comment by HipsterBrown** [comment]
@xdamman I’m away from my computer now. If we want this in Monday’s report feel free to add those changes and deploy.

**Comment by xdamman** [comment]
I've edited this Pull Request to use `formatCurrency`, `markdown` and added `Added Funds` with the `opencollective` payment method (which is more really "moving funds" since it will also include collective to collective donations)

---
Connecting to postgres://127.0.0.1/opencollective_prod_snapshot
# Weekly activity summary
Excluding [OC team collective](https://opencollective.com/opencollective-company)

## Donations
  - STRIPE: 151 donations received (-5)
    * CAD CA$86 (+CA$86)
    * EUR €887 (-€1,663)
    * GBP £26 (-£455)
    * NZD NZ$30 (+NZ$30)
    * USD $27,798 (-$11,866)
  - PAYPAL: 16 paypal donations received (+3)
    * USD $617 (+$337)
  - MANUAL: 18 donations received (-19)
    * CAD CA$86 (+CA$86)
    * EUR €565 (+€565)
    * USD $22,510 (+$22,510)

## Expenses
  - 36 paid expenses (-42)
    * CAD CA$273  (-CA$1,014)
    * EUR €234  (-€1,666)
    * USD $12,122  (-$8,889)
  - 6 pending expenses
    * EUR €2,638
    * USD $3,968
  - 18 approved expenses
    * EUR €619
    * USD $24,312
  - 6 rejected expenses
    * USD $53,048

## Collectives
  - 112 active collectives (-16)
  - 22 new collectives (+4):
    * [bundlephobia](https://opencollective.com/bundlephobia) (open source)
    * [adi yogpeeth](https://opencollective.com/adiyogpeeth1) (C-corp,other)
    * [GENNA Alliance](https://opencollective.com/genna-alliance) (C-corp,movement)
    * [United Vibrations](https://opencollective.com/united-vibrations) (C-corp,coop)
    * [BaseConverter](https://opencollective.com/baseconverter) (C-corp,studentclub)
    * [netassonetwork](https://opencollective.com/netassonetwork) ()
    * [standard-readme](https://opencollective.com/standard-readme) (open source)
    * [Skitch](https://opencollective.com/skitchnz) (C-corp,movement)
    * [Larp House](https://opencollective.com/larp-house) (larp)
    * [Voodoo Systems](https://opencollective.com/voodoosystems) (C-corp,other)
    * [Cascadian Integration Meta-Cooperative](https://opencollective.com/cascadian-integration-meta-cooperative) (C-corp,coop)
    * [Vancouver Technology Talents Networking @ 9CAT](https://opencollective.com/9cats) (entrepreneurship,professional-networking,decentralized-systems,cryptocurrency,blockchain,bitcoin,ethereum,eos,ai,machine-learning,internet-of-things,web,mobile-technology,startup-businesses,smallbiz,C-corp,meetup)
    * [gauth](https://opencollective.com/gauth) (open source)
    * [cheton](https://opencollective.com/cheton) (javascript)
    * [Testing service](https://opencollective.com/testing-service) (C-corp,association)
    * [Tech for Good New Zealand](https://opencollective.com/techforgoodnz) (meetup,tech4good,nonprofit,charity,technology,marketing,digital)
    * [3D-Druck-Community](https://opencollective.com/3d-druck-community) (C-corp,movement)
    * [easy-php](https://opencollective.com/easy-php) (open source)
    * [Ant Design](https://opencollective.com/antdesignui) (react,ui,design,enterprise,background)
    * [Docusaurus](https://opencollective.com/docusaurus) (open source)
    * [emotion](https://opencollective.com/emotion) (open source)
    * [JTAppleCalendar](https://opencollective.com/jtapplecalendar) (open source)

**Pull request merged**